adiachenko/src/components/Film/index.tsx

```diff
@@ -88,4 +107,4 @@ const Film = ({film}: FilmsProps) => {
   )
 };
 
-export default Film;
+export default React.memo(Film);
```

Comment:
I have, probably, mislead you while making my comments about `useMemo` related to this particular line. Probably, been a bit biased by constant talks about `useMemo` in our chats and didn't notice it's actually `React.memo` which is perfectly valid in this particular case.